admin/vagrant.py

```diff
@@ -13,10 +15,25 @@
 
 import flocker
 
-from admin.runner import run
 from admin.release import make_rpm_version
 
 
+def run(command, **kwargs):
```

Comment:
This is the same as `subprocess.call`, so I'm not sure if it worth expanding the name.